sql/core/src/test/resources/sql-tests/inputs/pgSQL/numeric.sql

```diff
@@ -0,0 +1,1095 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- NUMERIC
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/numeric.sql
+--
+
+CREATE TABLE num_data (id int, val decimal(38,10)) USING parquet;
```

Comment:
`decimal(38,10)` vs `numeric(210,10)`? Is this difference covered by SPARK-28318? Could you add a JIRA ID before this line?

Reply:
OK. Added it.